todo_app/backend.py

```diff
@@ -0,0 +1,46 @@
+import database
+import random
+import time
+
+
+itemList = []
+
+def mongodb_getcollections():
+    itemList.clear()
+    dbname = database.mongodb_connection()
+    item_col = dbname[database.MONGOCOLLECTION]
+    for i in item_col.find():
+        itemList.append(i)
+
+def mongodb_addcollection(taskName,taskDesc,taskStatus):
+    item_Number = f'ItemNo_{random.randrange(1,1000000)}'
```

Comment:
There's no issue with handling this as your own custom GUID, but worth noting that Mongo assigns a field `_id` automatically that functions as a unique ID, so we could take advantage of that instead